hack/crds-sync.sh

```diff
@@ -0,0 +1,32 @@
+ #!/usr/bin/env bash
+
+set -euo pipefail
+
+# map names of CRD files between the vendored openshift/api repository and the ./install directory
+CRDS_MAPPING=( "v1/0000_80_containerruntimeconfig.crd.yaml:0000_80_machine-config-operator_01_containerruntimeconfig.crd.yaml"
+               "v1/0000_80_kubeletconfig.crd.yaml:0000_80_machine-config-operator_01_kubeletconfig.crd.yaml"
+               "v1/0000_80_machineconfig.crd.yaml:0000_80_machine-config-operator_01_machineconfig.crd.yaml"
+               "v1/0000_80_machineconfigpool.crd.yaml:0000_80_machine-config-operator_01_machineconfigpool.crd.yaml" ) 
+                #TODO(jkyros): 0000_80_machine-config-operator_02_containerruntimeconfig.crd.yaml)
```

Comment:
Is there anything blocking us to copy this from vendor/ ?

Reply:
Nah, I just hadn't tried it yet. Looks like it works, I took this whole script out.